noip.sh

```diff
@@ -3,12 +3,22 @@
 SOURCE_CONF=/config/noip.conf
 GENERATED_CONF=/config/no-ip2.generated.conf
 
-# Search for custom config file, if it doesn't exist, copy the default one
```

Comment:
There's no need to create the file at all with your implementation. Just check to make sure the env vars are defined, then use them below. Remove the ". /tmp/noip.conf" line.